engines/sci/parser/vocabulary.cpp

```diff
@@ -188,6 +188,45 @@ bool Vocabulary::loadParserWords() {
 		seeker += 3;
 	}
 
+	// Additional translated words
+	{
+		resource = _resMan->findResource(ResourceId(kResourceTypeVocab, VOCAB_RESOURCE_SCI0_LOC_VOCAB), 0);
```

Comment:
This should also have a language check instead of always being triggered by VOCAB.001. There is a lot of weird SCI out there in the world; it's easy to imagine this getting accidentally triggered.

Reply:
Added check for russian language